testdata/ecosystem/pipelines/cache-stepactions-python.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: demo-pipeline-tekton-caches-python
+spec:
+  params:
+    - name: repo_url
+      type: string
```

Comment:
`type: string` is not necessary, it's a default value so we usually don't specify it explicitly.

Reply:
still unresolved